_tuning-your-cluster/availability-and-recovery/workload-management.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Workload management
+nav_order: 60
+has_children: false
+parent: Availability and recovery
+---
+
+# Workload management
+
+Workload management is a mechanism that allows administrators to organize queries into distinct groups, referred to as _query groups_. These query groups enable admins to limit the cumulative resource usage of each group, ensuring more balanced and fair resource distribution between them. This mechanism provides greater control over resource consumption, so that no single group can monopolize cluster resources at the expense of others.
+
+## Query group
+
+A query group is a logical construct designed to manage search requests within defined virtual resource limits. The query group service tracks and aggregates resource usage at the node level for different groups, enforcing restrictions to ensure that no group exceeds its allocated resources. Depending on the configured containment mode, the system can limit or terminate tasks that surpass these predefined thresholds.
+
+Since the definition of a query group is stored in the cluster state, these resource limits are enforced consistently across all nodes in the cluster.
```

Comment:
```suggestion
Because the definition of a query group is stored in the cluster state, these resource limits are enforced consistently across all nodes in the cluster.
```